src/util.cpp

```diff
@@ -418,6 +418,7 @@ bool SoftSetArg(const std::string& strArg, const std::string& strValue)
     if (mapArgs.count(strArg))
         return false;
     mapArgs[strArg] = strValue;
+    _mapMultiArgs[strArg].push_back(strValue);
```

Comment:
I think it is confusing for a "Set" function to add something to a list. If anything it should replace it.
What do you need this for, specifically?
In the past I've defined [SoftSetMultiArg](https://github.com/laanwj/bitcoin/commit/1024183eff629b758745aa7ebf9329d987179644) to be able to define a whole list argument at once.